integrations/build_system/cmake/cmake_find_package_multi_generator.rst

```diff
@@ -27,7 +27,13 @@ Usage
     $ conan install . -g cmake_find_package_multi -s build_type=Release
 
 These commands will generate several files for each dependency in your graph, including a ``XXXConfig.cmake`` that can be located
-by the CMake `find_package(XXX) <https://cmake.org/cmake/help/v3.0/command/find_package.html>`_ command, with XXX as the package name.
+by the CMake `find_package(XXX CONFIG) <https://cmake.org/cmake/help/v3.0/command/find_package.html>`_ command, with XXX as the package name.
+
+.. note::
```

Comment:
Use the important box (red color, more evident) https://github.com/conan-io/docs#text-boxes
```suggestion
.. important::
```